tests/migration/Core/V6_7/Migration1752219159AddLanguageActiveTest.php

```diff
@@ -0,0 +1,200 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Tests\Migration\Core\V6_7;
+
+use Doctrine\DBAL\Connection;
+use PHPUnit\Framework\Attributes\CoversClass;
+use PHPUnit\Framework\TestCase;
+use Shopware\Core\Defaults;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Test\TestCaseBase\KernelLifecycleManager;
+use Shopware\Core\Framework\Uuid\Uuid;
+use Shopware\Core\Migration\V6_7\Migration1752219159AddLanguageActive;
+use Shopware\Core\System\Language\LanguageDefinition;
+use Shopware\Core\System\Locale\LocaleDefinition;
+
+/**
+ * @internal
+ */
+#[Package('discovery')]
+#[CoversClass(Migration1752219159AddLanguageActive::class)]
+class Migration1752219159AddLanguageActiveTest extends TestCase
+{
+    private const TEST_LANGUAGES = [
+        'български език',
+        'Čeština',
+        'Dansk',
+        'Ελληνικά',
+        'Español',
+        'Français',
+        'हिन्दी',
+        '한국어',
+        'Nederlands',
+        'Norsk',
+        'Polski',
+        'Türkçe',
+        'Українська',
+        'Tiếng Việt Nam',
+    ];
+
+    private const TEST_INACTIVE_LANGUAGE_NAMES = [
+        '한국어',
+        'Nederlands',
+        'Tiếng Việt Nam',
+    ];
+
+    private const PACK_LANGUAGE_ENTITY_NAME = 'swag_language_pack_language';
+
+    private readonly Connection $connection;
+
+    private readonly Migration1752219159AddLanguageActive $migration;
+
+    protected function setUp(): void
+    {
+        $this->connection = KernelLifecycleManager::getConnection();
+        $this->migration = new Migration1752219159AddLanguageActive();
+
+        $tryStatements = [
+            'ALTER TABLE `language` DROP COLUMN `active`;',
+            \sprintf('DROP TABLE `%s`;', self::PACK_LANGUAGE_ENTITY_NAME),
```

Comment:
the table should be removed again after the tests

Reply:
Won't they be rolled back anyway?